frontend/src/sections/Hero/Hero.tsx

```diff
@@ -40,7 +40,7 @@ const HeroSection = () => {
                 width={532}
                 height={159}
                 alt='logo'
-                className='mx-auto'
```

Comment:
Make sure the content is centered

<img width="1375" alt="Screenshot 2024-08-08 at 12 06 29 AM" src="https://github.com/user-attachments/assets/26a90086-c2b2-4a1b-bec5-fb7605a9de95">